### PR TITLE
Deprecate swipeDown/Up, Reduce Verbosity, Update TestNG, and Update Java Client

### LINKED ISSUE #48: Remove swipeX methods as they swipe the wrong direction
All of the methods that are a simple "swipeX()" (e.g. swipeUp, swipeDown, swipeLeft, swipeRight) actually use the opposite SwipeElementDirection, which is just confusing.

I'd like to replace all these methods with a simple `swipe(SwipeElementDirection direction)` method

### LINKED ISSUE #150: Update TestNG to 7.1.0 Release
Update TestNG to 7.1.0 Release

### PR DESCRIPTION
**Pull request by mrk-han**
- Create scrollDown/Up

### Related Issue

closes #48 
closes #150 

### Description

Add deprecate annotations to original swipes, and create new methods scrollUp and scrollDown

### Checklist

* [x] Have you added an explanation of what your changes do?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: src/main/java/com/joss/conductor/mobile/Locomotive.java [hunk 1]
```diff
@@ -580,26 +580,51 @@ public Locomotive repetitiveCenterSwipe(SwipeElementDirection direction, int swi
     //endregion Generic swipes
 
     //region Directional swipes
+
+    @Deprecated
```
Comment (aaron-goff) [inline src/main/java/com/joss/conductor/mobile/Locomotive.java]: Nice to have: information about the new methods, i.e. "use `scrollDown() to achieve this behavior`

**Comment by aaron-goff** [comment]
Is there a reason that the swipeLeft/Right methods are not deprecated with new methods? Jeff mentions them as being incorrect in #48.

**Comment by mrk-han** [comment]
@aaron-goff 

Good call on the swipe right/left

File: src/main/java/com/joss/conductor/mobile/Locomotive.java (hunk 1 above)
Comment (mrk-han) [reply src/main/java/com/joss/conductor/mobile/Locomotive.java]: Fixed in fe7bf09709c1f8e139ed6bf324b6802244aecc35

**Comment by mrk-han** [comment]
> Is there a reason that the swipeLeft/Right methods are not deprecated with new methods? Jeff mentions them as being incorrect in #48.

Done in fe7bf09709c1f8e139ed6bf324b6802244aecc35

**Pull request merged**